Makefile

```diff
@@ -26,3 +26,16 @@ purge:                      ## Clean cache and leftovers. Please run this when s
 	git reset --hard && git clean -xdff
 	git submodule update
 	git submodule foreach 'git reset --hard && git clean -xdff'
+
+fb:                         ## Creates feature build branch.
+  # Usage: make fb mainBranch=PMM-2.0 featureBranch=PMM-XXXX-name submodules="pmm pmm-managed"
+	git checkout $(mainBranch)
+	make purge
+	git pull origin $(mainBranch)
+	git checkout -b $(featureBranch)
+	$(foreach submodule,$(submodules),git config -f .gitmodules submodule.$(submodule).branch $(featureBranch);)
+	make submodules
+	git add .gitmodules
+	$(foreach submodule,$(submodules),git add sources/$(submodule);)
+	git commit -m "$(shell awk -F- '{print $$1 FS $$2}' <<< $(featureBranch)) Update submodules"
+	git push origin $(featureBranch)
```

Comment:
I suggest excluding the push step from this target. When I update FB typically I use `git push --force ...`.
Also, it's better to check what you are going to push before actually doing that.